docs/chapter1/section3/src/4-1_ping.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+func main() {
+	e := echo.New()
+
+	e.GET("/ping", func(c echo.Context) error { return c.String(http.StatusOK, "pong\n") })
+	e.Logger.Fatal(e.Start(":<ポート番号>"))
```

Comment:
ポート番号8080で固定しちゃってよさそう(去年までは共用サーバー使ってたから一人一人違う番号にしないといけなくてこういう表記にしていたので)

もしくは他の章と統一して

```suggestion
	e.Logger.Fatal(e.Start(":{ポート番号}"))
```